### ISSUE TITLE
Add arabic

### ISSUE THREAD

**Issue by Osama465** [LNReader/lnreader#1362]
### Describe your suggested feature

When I read a novel in Arabic the punctuations are missed up because of the app being English
So could you please add Arabic as an app language so the punctuations be normal.
And this problem isn't from the site, I checked. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by CD-Z** [comment]
Yeah, should be done in the future, but for now you can add this to your custom CSS to fix this problem. 
```css
body {direction: rtl; text-align: right}
```

**Comment by Osama465** [comment]
Tysm!
